domains/gizmora2.json

```diff
@@ -0,0 +1,12 @@
+{
+    "description": "Personal developer profile/blog",
+    "repo": "https://github.com/GizmoRA2/me",
+    "owner": {
+        "username": "GizmoRA2",
+        "email": "gizmora2@gmail.com"
+    },
+    "record": {
+        "A": "125.236.202.192",

```

Comment:
Please finish your website.

Reply:
My apologies,
The website is available at that address, however it's shared hosting so it will only be available if your using the domain name when pointing to it.
I have uploaded the contents of the site as it will appear on the github link
Thanks for your patience

Reply:
Is it on a port?